### PR TITLE
Use infix style to apply plugins

### PR DESCRIPTION
**Pull request by Goooler**
It works for both Groovy and Kotlin dsl.

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -12,6 +12,7 @@ indent_size = 4
 indent_style = space
 indent_size = 2
 
-[*.md]
+[{*.md,mdx}]
 indent_style = space
 indent_size = 4
+trim_trailing_whitespace = false
```
Comment (Goooler) [inline .editorconfig:18]: Maybe we should just apply this for `*.md` only.

Comment (3flex) [reply .editorconfig:18]: Is this required for this change?

Comment (Goooler) [reply .editorconfig:18]: Actually nope. I edited these files on Visual Studio Code for the Web, formats changed, so I added this rule.

**Review by cortinico — APPROVED** [review]

File: .editorconfig (hunk 1 above)
Comment (3flex) [reply .editorconfig]: Thanks, I've reverted that commit in this PR.

**Review by 3flex — APPROVED** [review]

**Pull request merged**